app/script.js

```diff
@@ -285,12 +293,80 @@ let quote = {
     }
 }
 
+let focus = {
+    taskChecked: localStorage.getItem("main-focus-check"),
+    task: localStorage.getItem("main-focus"),
+    drawFocus: function () {
+        if (localStorage.getItem("username") !== null) {
+            if (focus.task == null || focus.task =='') {
+                $("#main-focus").html(`
+                <form id="main-focus-form">
+                    <label>What is your main focus (for) today?</label>
+                    <input type="text" name="focus" id="main-focus-value" placeholder="eg: cooking">
+                </form>`);
+            } else {
+                $("#main-focus").html(`<input type="checkbox" id='my-focus' value=" ` + focus.task + `"><label id='focusCheck'></label><span id='editfocus' contenteditable >` + focus.task + `</span><span id='main-focus-delete'><img id="main-focus-icon" src="img/delete-icon.png"/></span>`);
+            }
+            focus.ifChecked();
+        };
+        focus.getTask();
+        focus.editTask();
+        focus.deleteTask();
+
+        $('#focusCheck').click( function(e){
+            focus.taskChecked = !focus.taskChecked;
+             localStorage.setItem("main-focus-check", focus.taskChecked);
+             focus.drawFocus();
+
+        });
+
+    },
+
+    ifChecked: function() {
+        let check = localStorage.getItem("main-focus-check");
+        if (check == 'true') {
```

Comment:
Also always use three = (===), except when you explicitly want the implicit coersion